packages/lesswrong/components/seasonal/HomepageCommunityMap.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useUserLocation } from '../../lib/collections/users/helpers';
+import { forumTypeSetting } from '../../lib/instanceSettings';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import { useCurrentUser } from '../common/withUser';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+
+  }
+});
+
+export const HomepageCommunityMap = ({classes}: {
+  classes: ClassesType,
+}) => {
+  const { CommunityMapWrapper } = Components
+
+  const currentUser = useCurrentUser()
+  const currentUserLocation = useUserLocation(currentUser)
```

Comment:
This will ask the user to share their browser location when they load the home page (if we don't have a location stored for them already). I'm not sure if you would consider that too intrusive to have on the home page - if so, you can use the `dontAsk` flag here (ex. we use that for the events list in the sidebar).

Reply:
Thanks. Ruby and I chatted and decided this was okay to have for two weeks – it's a bit more intrusive, but there's an obvious special event going on that actually warrants the location sharing.